eslintrc.json

```diff
@@ -43,6 +44,8 @@
     "react/no-danger": [0],
     "react/no-string-refs": [0],
     "react/no-unescaped-entities": [0],
+    "react/no-unused-prop-types": [2, {"skipShapeProps": true}],
```

Comment:
How come we `skipShapeProps`?

Reply:
It allows people to describe the shape of something they're expecting without necessarily having to use everything they're expecting. Happy to remove this if eveyone is happy.

Reply:
Commenting here for completeness, @thatguynamedandy explained that eslint has issues confirming confirming the properties mentioned within `shape` are used when destructuring is used to access them.